.circleci/config.yml

```diff
@@ -157,10 +157,6 @@ workflows:
       - util_job:
           name: Code Linting
           steps:
-            - run:
```

Comment:
I think dropping the linter is a mistake, as I know from experience that many people use Emacs, but don't run the necessary formatting commands in it. I constantly get PRs with misaligned code and `cljfmt` has been instrumental in warding those off. Not a big deal, but I'd rather stay consistent across all of our projects in terms of lint checks.

Reply:
Aight, I dropped the commit.

Was going to suggest we turn it off temporarily, as I use aggressive indent mode and ~~fuck up~~ improve the indentation constantly.  Then it occurred to me that it would be some work to turn it back on again, after the holidays when I'll have little time to work on refactor-nrepl 😞